project-koku/masu
# Updating OCP fixed rate charge calculation with latest db schema

Updating masu fixed rate charge calculation to handle provider-specific rates and other changes to accommodate latest db schema.

**Testing**
1. Add an OCP provider upload data to upload service
2.  Add memory and cpu rates in koku
3. Download/Ingest/Process from masu
4. Confirm Koku API charge information is present

**Test Results**
[masu_fixed_rate_db_update_ut.txt](https://github.com/project-koku/masu/files/2581639/masu_fixed_rate_db_update_ut.txt)
